tests/kernel/run_kselftests.pm

```diff
@@ -111,3 +111,79 @@ sub post_run_hook {
 }
 
 1;
+
+=head1 Discussion
+
+This module executes Linux Kernel Selftests (kselftests) inside openQA.
+It supports running tests either from the in-tree git repository or from
+the packaged kselftest RPMs provided by OBS/IBS.
+
+The module groups tests by a collection, as listed in
+F<kselftest-list.txt>, and allows selecting individual tests, skipping
+tests, and injecting custom environment variables into the test harness.
+
+Test results are collected from KTAP output produced by the
+F<run_kselftest.sh> harness and exported into the openQA result
+directory. When multiple tests are executed, per-test logs are enabled
+automatically.
+
+A serial console stamp is written before and after the test run to
+detect hangs or kernel crashes. A global timeout is applied to the
+overall test run, while an optional per-test timeout can be used to
+override the default 45-second limit built into the kselftest harness.
+
+=head1 Configuration
+
+The kselftest module is configured through openQA test variables:
+
+=head2 KSELFTEST_COLLECTION (required)
+
+Specifies the name of the kselftest collection to run, as reported by:
+
+  run_kselftest.sh --list
+
+=head2 KSELFTEST_TESTS
+
+Optional list of individual tests to run from within the selected
+collection. When not provided, the entire collection is executed.
+
+=head2 KSELFTEST_SKIP
+
+Optional list of tests that should be skipped. This is applied after
+KSELFTEST_TESTS, so it can exclude tests even when running a full
+collection.
+
+=head2 KSELFTEST_FROM_GIT
+
+If set, kselftests are installed from a kernel git tree instead of using
+packaged RPMs. Allows to point to C<KERNEL_GIT_TREE>. Defaults to the
+upstream tree: C<torvalds/linux.git>.
+
+=head2 KSELFTEST_RUNNER
+
+Overrides the default runner command. Useful for debugging or running
+custom wrappers.
+
+=head2 KSELFTEST_TIMEOUT
+
+Applies a global timeout (in seconds) to the entire kselftest run. If
+the tests do not complete within this time, the module fails.
+
+=head2 KSELFTEST_TEST_TIMEOUT
+
+Optional per-test timeout passed to F<run_kselftest.sh> via the
+C<--override-timeout> argument. This overrides the default kselftest
+per-test timeout (typically 45 seconds). Useful for long-running tests.
+
+=head2 KSELFTEST_ENV
+
+Optional string containing a list of environment variables to inject
+into the kselftest runner.
```

Comment:
Maybe here too?
```KSELFTEST_ENV="KSELFTEST_TEST_PROGS_CPUV4_ARGS='-v -t verifier'"```

Would also be nice to have a sentence detailing the usefulness. E.g. "Useful to pass down arguments to specific tests via `run_kselftest.sh`. More details in  `tools/testing/selftests/kselftest/runner.sh` file upstream."